src/electionguard_tools/helpers/__init__.py

```diff
@@ -61,6 +61,7 @@
     "accumulate_plaintext_ballots",
     "export",
```

Comment:
Since export was renamed to export_record, do we still need the export name here?

Reply:
This was the name of the module. These are automade